content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+
+<!-- body -->
+
+## Nodo a Control Plane
+
+Kubernetes tiene un patrón de API "hub-and-spoke" (centro y radios). Todo uso de la API desde los nodos (o los pods que ejecutan) termina en el servidor API. Ninguno de los otros componentes del plano de control está diseñado para exponer servicios remotos. El servidor API está configurado para escuchar conexiones remotas en un puerto HTTPS seguro (típicamente el 443) con una o más formas de
+[autenticación](/docs/reference/access-authn-authz/authentication/) de cliente habilitadas.
+
+Se debe habilitar una o mas formas de  [autorizacion](/docs/reference/access-authn-authz/authorization/), especialmente si se permiten [request anonimos](/docs/reference/access-authn-authz/authentication/#anonymous-requests)
+o [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+Los nodos debe ser aprovisionados con el {{< glossary_tooltip text="certificado" term_id="certificate" >}} raíz público del cluster para que puedan conectar de manera segura al API server junto con credenciales de cliente válidas. Un buen enfoque es que las credenciales del cliente proporcionadas al kubelet estén en forma de un certificado de cliente. Consulta [kubelet TLS bootstrapping] (/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/) para la provisión automatizada de certificados de cliente del kubelet.
+
+{{< glossary_tooltip text="Pods" term_id="pod" >}} que deseen conectarse al servidor API pueden hacerlo de manera segura aprovechando una cuenta de servicio para que Kubernetes inyecte automáticamente el certificado raíz público y un token portador válido en el pod cuando este es instanciado. El servicio  `kubernetes` (en el namespace `default`)  está configurado con una dirección IP virtual que es redirigida (a través de {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}) al punto final HTTPS en el servidor API.
+
+
+Los componentes del control plane se comunican con el servidor API a través del puerto seguro.
+
+Como resultado, el modo de operación predeterminado para las conexiones desde los nodos y el pod que se ejecutan en los nodos hacia el plano de control está asegurado por defecto y puede funcionar sobre redes no confiables y/o públicas.
+
+## Control plane al nodo
+
+Existen dos formas de comunicación primarias desde el control plane (el servidor API) hacia los nodos. El primero es desde el servidor API hasta el proceso {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} que se ejecuta en cada nodo del clúster. El segundo es desde el servidor API hacia cualquier nodo, pod o servicio a través de la funcionalidad de proxy del servidor API.
```

Comment:
We have used "plano de control" as the translation for "control plane" in other Spanish articles. For instance, the official concept meaning translation:

https://kubernetes.io/es/docs/concepts/overview/components/#componentes-del-plano-de-control

Please use "Plano de Control" to keep consistency.

```suggestion
Existen dos formas de comunicación primarias desde el plano de control (el servidor API) hacia los nodos. El primero es desde el servidor API hasta el proceso {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} que se ejecuta en cada nodo del clúster. El segundo es desde el servidor API hacia cualquier nodo, pod o servicio a través de la funcionalidad de proxy del servidor API.
```